src/compiler/checker.ts

```diff
@@ -39975,6 +40031,10 @@ namespace ts {
                 }
             }
 
+            if (isMetaProperty(node.parent) && node.parent.keywordToken === node.kind) {

```

Comment:
Just to double check - is `keywordToken` a `SyntaxKind` and not a `Token<SyntaxKind>`? I've messed that up before.

Reply:
Yeah, it's a bit ambiguous. It's a `SyntaxKind`:

```ts
export interface MetaProperty extends PrimaryExpression {
    readonly kind: SyntaxKind.MetaProperty;
    readonly keywordToken: SyntaxKind.NewKeyword | SyntaxKind.ImportKeyword;
    readonly name: Identifier;
}
```